diagnostic_remote_logging/include/diagnostic_remote_logging/influx_line_protocol.hpp

```diff
@@ -0,0 +1,158 @@
+/*********************************************************************
+ * Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2009, Willow Garage, Inc.
```

Comment:
Please change

Reply:
This should be the copyright holder. And that is you or your company, depending on how you handle these things.